### PR TITLE
Added support for different icons - take 2

### PR DESCRIPTION
**Pull request by adi-miller**
Specifically, I wanted to be able to use the fa-twitter-square icon. So added support to just specify the name of the icon. If missing from the configuration, the default is fa-twitter.

### CODE REVIEW

**Comment by adi-miller** [comment]
Ping...?

**Comment by ngnijland** [comment]
Sorry @adi-miller, I layed sick in bed all week. I will review your PR now, thanks for the contribution!

File: MMM-social-counter.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ Module.register('MMM-social-counter', {
     this.twitter = this.config.twitter;
     this.updatesEvery = this.config.updatesEvery;
     this.size = this.config.size;
+    this.icon = this.config.icon || 'fa-twitter';
```
Comment (ngnijland) [inline MMM-social-counter.js]: I'd rather have all default values in one overview at the `defaults` object at the top of this file. Can you move this please?

File: README.md [hunk 2]
```diff
@@ -80,4 +80,5 @@ The following configurations are available:
 Config                | Type                       | Default value | Description
 :---------------------|:---------------------------|:--------------|:------------
 `size`                | `small \| medium \| large` | `medium`      | The size of the counter
-`updatesEvery`        | `number`                   | `10`          | The amount of seconds between each follower count update
+`icon`                | `string                    | `fa-twitter`  | The font name (from FontAwesome) to display next to the follower count, e.g. fa-twitter-square
```
Comment (ngnijland) [inline README.md]: I think this is a typo?

`The font name` -> `The icon name`

**Review by ngnijland — CHANGES_REQUESTED** [review]

**Comment by adi-miller** [comment]
Submitting a new PR with fixes.

**Pull request closed without merging**